### DISCUSSION TITLE
GTK3 apps maximize and close buttons not showing the right title bar button theme.

### DISCUSSION THREAD

**Discussion by kdgnlx** [GradienceTeam/Gradience#720] [General]
I applied a theme with "whitesur" style title bar button. In GTK4 apps everything works fine but for firefox and any gtk3 app I can only see the color theme but not the themed title bar button. Only the adwaita default ones. 
![Screenshot from 2023-02-14 12-39-13](https://user-images.githubusercontent.com/96071375/218731257-c766155d-84c7-433f-834b-3f61af92b130.png)
![Screenshot from 2023-02-14 12-55-34](https://user-images.githubusercontent.com/96071375/218731286-d26e0a35-30a1-4e75-9436-65f3ab618998.png)

I followed the steps to install Gtk3theme.adw-gtk3 and gave proper permissions to flatpak apps.
https://github.com/lassekongo83/adw-gtk3

**Comment by tfuxu** [comment]
@kdgnlx 
You need to also add custom CSS for titlebar buttons to GTK3 output CSS in `Custom CSS` textview.
![Screenshot from 2023-02-17 00-39-55](https://user-images.githubusercontent.com/73042332/219512570-98875428-430d-40cf-9b22-544cf37d8b47.png)
